sdk/src/cameras/chicony-006/camera_chicony_006.cpp

```diff
@@ -338,6 +338,11 @@ aditof::Status CameraChicony::requestFrame(aditof::Frame *frame,
     uint16_t *frameDataLocation;
     frame->getData(FrameDataType::FULL_DATA, &frameDataLocation);
 
+    if (frame == nullptr){
```

Comment:
the validation should be done **before** the usage of the pointer.

Reply:
Fixed